src/pages/MyPage/Overview/InvestContent.js

```diff
@@ -1,45 +1,71 @@
 import React from 'react';
 import styled from 'styled-components';
 import { Doughnut } from 'react-chartjs-2';
+import { makeChartData, CHART_OPTION } from './ChartOption';
 
-function InvestContent({ investStatus }) {
-  const expData = {
-    datasets: [
-      {
-        labels: ['투자중', '투자완료', '손실'],
-        data: [
-          investStatus.invest,
-          investStatus.complete,
-          investStatus.insolvency,
-        ],
-        borderWidth: 0,
-        backgroundColor: ['#6C3AD3', '#DEDEF4', '#D2D2D2'],
-      },
-    ],
+function InvestContent({ investStatus, currData }) {
+  //받아온 데이터의 타이틀과 화면에 보여줄 타이틀 매칭시키기 위한 객체
+  const nameMatch = {
+    totalInvest: '누적 투자 원금',
+    complete: '투자완료',
+    delay: '상환지연',
+    invest: '투자중',
+    loss: '손실',
+    normal: '정상',
+    overdue: '연체',
+    nonperform: '부실',
   };
-  const expOption = {
-    options: {
-      cutoutPercentage: 30,
-    },
+
+  // 구조분해할당
+  const {
+    totalInvest,
+    complete,
+    delay,
+    invest,
+    loss,
+    normal,
+    overdue,
+    nonperform,
+  } = investStatus;
+
+  //전체투자액와 투자중금액 데이터 나눠주기(탭으로 분리됨)
+  const state = {
+    total: { invest, complete, loss },
+    investing: { normal, delay, overdue, nonperform },
   };
+
+  // 선택된 탭에 따른 데이터 변경(디벨롭 할 수 있을지 궁금...)
+  const entries = currData
+    ? Object.entries(state.total)
+    : Object.entries(state.investing);
+
+  // map으로 돌려줄 리스트 생성
+  const ConvertToChartData = entries.map(([name, value]) => [
+    nameMatch[name],
+    value,
+  ]);
+
+  // 차트 데이터 생성
+  const chartData = makeChartData(entries);
+
   return (
     <InvestContentBox>
       <Chart>
-        <Doughnut data={expData} height={180} options={expOption} />
+        <Doughnut data={chartData} height={180} options={CHART_OPTION} />
+        <ChartTitile>
+          {currData ? '누적 투자 원금' : '투자 중 원금'}
+        </ChartTitile>
+        <ChartSubtitle>
+          {currData ? totalInvest.toLocaleString() : invest.toLocaleString()}원
+        </ChartSubtitle>
       </Chart>
       <ChartDesc>
-        <DescContent>
-          <div>투자중</div>{' '}
-          <span>{investStatus.invest.toLocaleString()}원</span>
-        </DescContent>
-        <DescContent>
-          <div>투자완료</div>{' '}
-          <span>{investStatus.complete.toLocaleString()}원</span>
-        </DescContent>
-        <DescContent>
-          <div>손실</div>
-          <span>{investStatus.insolvency.toLocaleString()}원</span>
-        </DescContent>
+        {ConvertToChartData.map((item, idx) => (
```

Comment:
```suggestion
        {ConvertToChartData.map(([title,price], idx) => (
```
구조분해 해서 쓰면 좀 더 깔끔하겠네요!

Reply:
오 생각을 못했습니다 감사합니다.